ocaml/tests/test_repository_helpers.ml

```diff
@@ -2755,7 +2755,8 @@ module GuidanceSetResortGuidancesTest = Generic.MakeStateless (struct
   let transform (kind, guidances) =
     guidances
     |> GuidanceSet.of_list
-    |> GuidanceSet.resort_guidances ~kind
+    |> GuidanceSet.resort_guidances
+         ~remove_evacuations:(kind = Guidance.Absolute)
```

Comment:
why does this deal with absolute guidances when the concept is being dropped from XAPI?

Reply:
It's still part of the update metadata.